pkg/codegen/testing/test/testdata/components-pp/dotnet/components.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using Pulumi;
+
+return await Deployment.RunAsync(() => 
+{
+    var exampleComponent = new Components.ExampleComponent("exampleComponent", new()
```

Comment:
Wheres the ExampleComponent source file?

Reply:
Was not generated because program test only generates the top-level file 😅

Reply:
@Frassle I've extended ProgramTest, it now understand how to generate multiple files (when `PULUMI_ACCEPT=1`) and asserting their contents in the default case ✅